rx-java2-gen/src/main/java/io/vertx/lang/reactivex/Helper.java

```diff
@@ -22,4 +44,16 @@ public static Class unwrap(Class<?> type) {
     }
     return type;
   }
+
+  /**
+   * Convert a handler for a generated reactive type to a handler for the corresponding core type.
+   */
+  public static <CORE, REACTIVE> Handler<CORE> convertHandler(Handler<REACTIVE> rxHandler, Function<CORE, REACTIVE> mapper) {
+    RxGen annotation = rxHandler.getClass().getAnnotation(RxGen.class);
+    if (annotation != null && rxHandler instanceof RxDelegate) {
+      RxDelegate rxDelegate = (RxDelegate) rxHandler;
+      return (Handler<CORE>) annotation.value().cast(rxDelegate.getDelegate());
```

Comment:
@vietj I'm not sure there is much interest in `annotation.value().cast()` here...

Maybe we could get away with `return (Handler<CORE>) rxDelegate.getDelegate();`